README.md

```diff
@@ -0,0 +1,175 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [:camera: screenshots](#screenshots)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+
+- [📝 License](#license)
+
+# 🏥 Houses Resrvation API <a name="about-project"></a>
+
+> The Houses Resrvation API allows users to create account and create new House and can reserved the House.
+
+### Frontend Repository
+  <a href="https://github.com/mercymugambi/frontend_house_reservations_react_app.git">Final capstone front end</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+- <a href="https://www.ruby-lang.org/es/">ROR</a>
+- <a href="https://www.postgresql.org/">Postgresql</a>
+- <a href="https://rspec.info/">Rspec</a>
+
+### :rocket: Front End API <a name="live-demo"></a>
+https://github.com/mercymugambi/frontend_house_reservations_react_app/
+
+
+  - <a href="">Live Link</a>
+
+### Key Features <a name="key-features"></a>
+
+- [x] Create user based on role
+- [x] Create house and view all houses
+- [x] Create reservations and view all reservations
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Command line or terminal
+- IDE such as VsCode.
+- Rails needs to be installed on your system.
+
+### Setup
+
+Clone this repository to your desired folder:
+Example command:
+
+```sh
+  git clone https://github.com/mercymugambi/backend_house_reservations_rails_app.git
+  cd full-stack-capstone-back-end
+
+```
+
+### Install
+
+Example command:
+
+```sh
+  cd full-stack-capstone-back-end
+  bundle install (install all dependencies)
+  rails db:create (to create database)
+  rails db:migrate (to migrate tables)
+
+```
+
+### Usage
+
+To run the project redirecting to the root directory of the project and run the following command:
+
+```sh
+ bin/rails server (to run web API)
+
+```
+
+### Test
+
+To test the web app run the following command:
+
+```sh
+ rspec spec ( to test the web API)
+
+```

```

Comment:
- [ ] You don't include any unit tests:

![image](https://github.com/mercymugambi/backend_house_reservations_rails_app/assets/57421823/50169dd8-419a-45d0-9231-0666dd76a6f2)

Therefore, kindly either add some or remove this section.